### PR TITLE
Resolves #984, removes dependency on 'build' folder existing

### LINKED ISSUE #984: Clean install grunt build cannot find config.json
Fresh install doesn't seem to be working:

```
$ grunt build
Loading "Gruntfile.js" tasks...ERROR
>> Error: Unable to read "/Users/Si1kIfY/dev/ucopea/build/course/config.json" file (Error code: ENOENT).
```

I'll try to remove / re-install and report back

### PR DESCRIPTION
**Pull request by brian-learningpool**
Also using path.join() now rather than string concatenation for folder names.

### CODE REVIEW

**Comment by oliverfoster** [comment]
+1

**Comment by moloko** [comment]
+1 fixes the original issue (tested on OS X 10.11.3) - but I do get the following in the output, anything to be concerned about?
```
Running "replace:dist" (replace) task
>> Unable to match 1 pattern, remember for simple matches (String) we are using the prefix @@ for replacement lookup.
>> 0 replacements in 1 file.
```

File: grunt/config/replace.js [hunk 1]
```diff
@@ -1,19 +1,46 @@
...
+            flatten: true,
+            src: [options.outputdir + '*.xml'],
+            dest: options.outputdir
+          }
+        ]
+      }
+    }
   }
+  
+  var configJson = grunt.file.readJSON(pathToConfig);
```
Comment (ryangadams) [inline grunt/config/replace.js]: Would it be better to wrap this line in a try catch and set configJson to an empty object on error? Is is possible to have a circumstance where a replacement task should still be executed with the data in courseJson even if configJson is missing?

Comment (oliverfoster) [inline grunt/config/replace.js]: or do files:[] ? i think the replace task still needs some configuration due to the way the tasks are queued.

**Comment by brian-learningpool** [comment]
@moloko, nothing to be concerned about but I have added `silent: true` now to suppress the logging here.

**Comment by oliverfoster** [comment]
+1

**Pull request merged**